substrate/frame/revive/src/limits.rs

```diff
@@ -47,7 +47,7 @@ pub const NUM_EVENT_TOPICS: u32 = 4;
 pub const DELEGATE_DEPENDENCIES: u32 = 32;
 
 /// Maximum size of events (including topics) and storage values.
-pub const PAYLOAD_BYTES: u32 = 512;
+pub const PAYLOAD_BYTES: u32 = 384;
```

Comment:
This seems unrelated. Why was this changed?

Reply:
Should have pointed out. The integrity checks fail with 2 seconds ref_time and payload bytes of 512 because the  `max_events_size < storage_size_limit` fails. I lowered this because I think we want 2s ref_time per block.

If there is a better solution or the integrity check is incorrect we should fix that, I'm open.

Reply:
On a parachain you only have 0.5s of execution time. With async backing its 1 second I think. But probably better to lower this ratio in order to future proof.

Reply:
Correction 2: https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-3---activate-async-backing